app/assets/javascripts/countdown.js

```diff
@@ -0,0 +1,38 @@
+function Countdown(trackingElements) {
+  this.trackingElements = trackingElements;
+  this.auction_end_time = trackingElements.auction_end_time.text();
+
+  this.ongoing = function(durationCalculator) {
```

Comment:
You should define these methods on the prototype: http://stackoverflow.com/questions/4508313/advantages-of-using-prototype-vs-defining-methods-straight-in-the-constructor